### DISCUSSION TITLE
ArduinoOTA on Atmel Studio 7

### DISCUSSION THREAD

**Discussion by talhayousuff** [JAndrassy/ArduinoOTA#96] [Q&A]
Hey jandrassy, ArduinoOTA has worked perfectly on Arduino with SIM800C, now I am trying to implement it on Atmel Studio. Do you have any support files or a method through which I can use ArduinoOTA on Atmel Studio 7?

**Comment by JAndrassy** [comment]
AVR without Arduino?

**Reply by talhayousuff** [reply]
I meant to say that ArduinoOTA has worked perfectly on Arduino IDE and now I am trying to implement it on the same microcontroller with Atmel Studio.

**Reply by JAndrassy** [reply]
yes, but Atmel Studio with Arduino support or without?
if with Arduino plugin, then what is the problem?
if without Arduino, then you have to port InternalStorageAVR.cpp and you will still need Optiboot and avrdude USB upload configured for Optiboot

**Reply by talhayousuff** [reply]
Thank you for your response, jandrassy. Let me try this.
